### DISCUSSION TITLE
Will updating HACS to 2.0.0 break this intgration?

### DISCUSSION THREAD

**Discussion by LenzGr** [StephanU/ha-talent-monitor#42] [General]
HACS released a major update to version 2.0.0, which includes breaking changes. From the "Breaking Changes" section of the [release notes](https://github.com/hacs/integration/releases/2.0.0):

> * The `sensor` platform has been removed

Will this upgrade break the TalentMonitor integration?

**Comment by StephanU** [comment]
I think this refers to the sensors which were previously published by HACS itself (like "last updated at"). The sensors of the integrations installed via HACS will not be removed.

**Reply by LenzGr** [reply]
OK, thanks for the confirmation! Indeed, updating to HACS 2.0.x did not break the integration.
